src/Examples/Flowsheets/CO2_adsorption_desorption/NETL_32D_solid_phase_thermo.py

```diff
@@ -208,14 +208,21 @@ def define_metadata(cls, obj):
                 # TODO - also use flow_mass state var if moving or bubbling fludized
                 # bed unit models are used
                 # "flow_mass": {"method": None, "units": "kg/s"},
-                "dens_mass_particle": {"method": None, "units": None},
                 "temperature": {"method": None, "units": "K"},
                 "mass_frac_comp": {"method": None, "units": None},
-                "mass_frac_comp_max": {"method": "_mass_frac_comp_max", "units": None},
                 "cp_mass": {"method": "_cp_mass", "units": "J/kg.K"},
                 "enth_mass": {"method": "_enth_mass", "units": "J/kg"},
             }
         )
+        
+        obj.define_custom_properties(
+            {
+                "dens_mass_particle": {"method": None,
+                                       "units": pyunits.dimensionless},
+                "mole_frac_comp_max": {"method": "_mole_frac_comp_max",
+                                       "units": pyunits.dimensionless},
```

Comment:
Was this change from mass_frac_comp_max to mole_frac_comp_max intended?

Reply:
There was no change - this was copied directly from where it was before.

Reply:
Really? I thought line 214 was removed so that it could be defined here as a custom property?

Reply:
Corrected - just a copy-and-paste error.